.env.example

```
@@ -0,0 +1,37 @@
+
+# Docker Compose PostgreSQL (컨테이너용)
+# Docker 컨테이너로 실행할 경우
+POSTGRES_DB=예시_데이터
+POSTGRES_USER=예시_데이터
+POSTGRES_PASSWORD=예시_데이터
+
+# Spring Datasource (로컬 개발용)
+# IDE에서 직접 실행할 경우
+SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/예시_데이터
+SPRING_DATASOURCE_USERNAME=예시_데이터
+SPRING_DATASOURCE_PASSWORD=예시_데이터
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SPRING_DATASOURCE_PASSWORD key should go before the SPRING_DATASOURCE_URL key

(UnorderedKey)